src/main/java/com/training/assignments/miniproject/EmployeeOperation.java

```diff
@@ -0,0 +1,20 @@
+package com.training.assignments.miniproject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EmployeeOperation {
+    static List<com.training.assignments.miniproject.Employee> employeeList = new ArrayList<>();
+
+    public void createEmployee(String name, String address, int salary, int id) {
+        employeeList.add(new com.training.assignments.miniproject.Employee(name, address, salary, id));
+    }
+
+    public List<com.training.assignments.miniproject.Employee> getEmployee() {
+        return employeeList;
+    }
+
+    public void deleteEmployee(int index) {
```

Comment:
lets overload this method with the below forms:
- `deleteEmployee(int id)` -> this would delete the employee based on id.
- `deleteEmployee(String name)` -> this would the first employee in the list matching with the provided name.

Also create tests for testing these scenarios